enterprise/server/remote_execution/containers/podman/podman_test.go

```diff
@@ -142,5 +145,47 @@ func TestIsImageCached(t *testing.T) {
 			assert.NoError(t, err)
 		}
 	}
+}
+
+func TestForceRoot(t *testing.T) {
+	rootDir := testfs.MakeTempDir(t)
+	testfs.MakeDirAll(t, rootDir, "work")
+	ctx := context.Background()
+	ctx, cancel := context.WithTimeout(ctx, 2*time.Minute)
+	defer cancel()
+	env := testenv.GetTestEnv(t)
+	env.SetAuthenticator(testauth.NewTestAuthenticator(testauth.TestUsers("US1", "GR1")))
+	cacheAuth := container.NewImageCacheAuthenticator(container.ImageCacheAuthenticatorOpts{})
+	image := "gcr.io/flame-public/test-nonroot:test-enterprise-v1.5.4"
+
+	cmd := &repb.Command{
+		Arguments: []string{"id", "-u"},
+	}
 
+	tests := []struct {
+		desc      string
+		forceRoot bool
```

Comment:
nit: you could add the expected UID here to avoid conditional logic in the test

Reply:
Done